tests/acceptance/resolve-guid-test.ts

```diff
@@ -49,22 +49,6 @@ module('Acceptance | resolve-guid', hooks => {
         sinon.stub(router, '_beforeTransition').returnsArg(0);
     });
 
-    test('User | Index', async assert => {
-        const user = server.create('user');
-
-        await visit(`/${user.id}`);
-
-        routingAssertions(assert, '--user', `/${user.id}`, 'guid-user.index');
-    });
-
-    test('File | Index', async assert => {
```

Comment:
Should we just skip this test for now, since this route is going to be re-implemented? Or is there something I'm missing here?

Reply:
It was a bit of a pain, because the typescript breaks as users don't have files, so if I commented out L61, then L63-65 don't work, since there wouldn't be a file. I figured it would be easy enough to recreate when the time comes.